tests/specs/publish/cloudflare_specifier/mod.ts

```diff
@@ -0,0 +1 @@
+import "cloudflare:sockets";
```

Comment:
Nitpick: maybe combine all these tests into one test with all the different specifiers in one file? It will help reduce integration test time.